.github/workflows/validate_build.yml

```diff
@@ -0,0 +1,32 @@
+name: validate_build
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  validate_build:
+    name: validate_build
+    runs-on:
+      - ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js latest
+      uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+        cache: npm
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Build and verify MAJC dist files
+      run: npm run build
+
+    - name: Install React Example dependencies
+      run: npm ci
+      working-directory: ./examples/react
+
+    - name: Build and verify Example React App
+      run: npm run build
```

Comment:
Not sure if it'd be overkill, but would it be helpful to run this build in the two different node versions like we do the unit tests?

Reply:
Since Node is only used here as part of the build system, I'm not sure that's necessary.